test/tempfile_test.py

```diff
@@ -72,8 +72,8 @@ def test_raylet_tempfiles():
     assert log_files == {
         "log_monitor.out", "log_monitor.err", "plasma_store.out",
         "plasma_store.err", "webui.out", "webui.err", "monitor.out",
-        "monitor.err", "redis-shard_0.out", "redis-shard_0.err", "redis.out",
-        "redis.err"
+        "monitor.err", "raylet_monitor.out", "raylet_monitor.err",
```

Comment:
nit: is it possible to avoid these strings? i.e., make these variables or something

Reply:
Maybe in some other PR